test/functional/editions_controller_test.rb

```diff
@@ -200,4 +200,66 @@ class EditionsControllerTest < ActionController::TestCase
       end
     end
   end
+
+  context "edit" do
+    setup do
+      @answer = FactoryBot.create(:answer_edition)
+    end
+
+    should "render the page ok" do
```

Comment:
Do we need this test? Isn't it just a simpler form of the one on line 47? We could then get rid of the "edit" context and just have the "#update" one.